Java Arraylist replaceAll()

CodedByCoffee opened the pull request:
<!--
👋 Hi, thanks for submitting a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->
Added documentation for the replaceAll() method in the ArrayList class as an .md file called replaceAll.md


### Issue Solved

<!--
Please use the following format(s) to link the issue numbers that your contribution addresses:
Closes #issue-number (where `issue-number` corresponds to the PRs' respective issue)
-->
Closes #4345

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

cigar-galaxy82 commented:
Hi, @CodedByCoffee can you please sign the CLA.

cigar-galaxy82 commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
```diff
@@ -0,0 +1,57 @@
+---
+Title: '.replaceAll()'
+Description: 'Replaces all occurrences of a given value ArrayList.'
```
```suggestion
Description: 'Replaces all occurrences of a given value in an ArrayList.'
```

cigar-galaxy82 commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
```diff
@@ -0,0 +1,57 @@
+  - 'paths/computer-science'
+---
+
+The **`.replaceAll()`** method of the `ArrayList` class takes an Operator as argument and applies it to every element in the List
```
```suggestion
The `.replaceAll()` method of the `ArrayList` class takes an operator as argument and applies it to every element in the list
```

CodedByCoffee replied:
Should we not keep the first replaceAll() occurence bold as per this guide? https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#writing-entries.
But maybe remove the backtick?

CodedByCoffee replied:
@cigar-galaxy82 See my questsion above ☝️

cigar-galaxy82 commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
```diff
@@ -0,0 +1,57 @@
+Subjects:
+  - 'A subject name'
+  - 'A second subject name'
+  - 'An nth subject name'
```
Two Subjects works for this entry

cigar-galaxy82 commented:
Hey @CodedByCoffee 

Thank you for contributing to Codecademy Docs! The entry is well written!

I've suggested a few changes modify them at your convenience

Thank you!

CodedByCoffee commented:
> Hi, @CodedByCoffee can you please sign the CLA.

I cannot find how I do that, where is the CLA and how do I sign it? 😅

cigar-galaxy82 commented:
Hi @CodedByCoffee don't worry about the CLA just make the changes sometimes our CLAassistant behave unpredictably

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=4642) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=4642) before we can accept your contribution.<br/><hr/>**GROUPINFRA\mattias.gradin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=4642) it.</sub>

mamtawardhani commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
```diff
@@ -0,0 +1,56 @@
+---
+Title: '.replaceAll()'
+Description: 'Replaces all occurrences of a given value in an ArrayList.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
```
```suggestion
---
Title: '.replaceAll()'
Description: 'Replaces all occurrences of elements in the ArrayList with a specified new value..'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Arrays'
  - 'Data Types'
  - 'Java'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---
```

mamtawardhani commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
```diff
@@ -0,0 +1,56 @@
+  - 'paths/computer-science'
+---
+
+The **`.replaceAll()`** method of the `ArrayList` class takes an Operator as argument and applies it to every element in the List
```
```suggestion
The **`.replaceAll()`** method in Java is used for transforming all elements of a list according to a specified unary operation, effectively updating each element in place with the result of the operation applied to it.
```

mamtawardhani commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
```diff
@@ -0,0 +1,56 @@
+
+## Syntax
+
+Applies `operator` to each element in ArrayList `myArrayList` and replaces the element with a new value
```
```suggestion
```

mamtawardhani commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
````diff
@@ -0,0 +1,56 @@
+
+Applies `operator` to each element in ArrayList `myArrayList` and replaces the element with a new value
+```pseudo
+myArrayList.replaceAll(operator);
````
```suggestion
arrayList.replaceAll(UnaryOperator<E> operator)
```

mamtawardhani commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
````diff
@@ -0,0 +1,56 @@
+```pseudo
+myArrayList.replaceAll(operator);
+```
+
````
Please describe the parameters in the following format:

- `arrayList`: parameter description
- `UnaryOperator<E> operator`: parameter description

mamtawardhani commented on content/java/concepts/array-list/terms/replaceAll/replaceAll.md:
````diff
@@ -0,0 +1,56 @@
+The output for this code would look like this:
+
+```shell
+Cities in uppercase: 
+[STOCKHOLM, LONDON, PARIS, BERLIN]
+Updated cities in lowercase: 
+[stockholm, london, paris, berlin]
````
```suggestion
Cities in uppercase: [STOCKHOLM, LONDON, PARIS, BERLIN]
Updated cities in lowercase: [stockholm, london, paris, berlin]
```

mamtawardhani reviewed (changes requested):
Hey @CodeByCoffee! Thank you for the contribution to Codecademy Docs! 😄 

I've requested some changes for this entry, could you please review and modify those at your earliest convenience? Thank you!

mamtawardhani commented:
Hey, @CodedByCoffee could you make the requested changes at your earliest convenience, so that this PR can be merged soon? Thanks 😄

mamtawardhani commented:
> [![CLA assistant check](https://camo.githubusercontent.com/fbf2b4a56b8f8dacb3fb4f7f3cf7877b8c27f9cfee8546ff24c8df94facca0f6/68747470733a2f2f636c612d617373697374616e742e696f2f70756c6c2f62616467652f6e6f745f7369676e6564)](https://cla-assistant.io/Codecademy/docs?pullRequest=4642) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=4642) before we can accept your contribution.**1** out of **2** committers have signed the CLA.✅ mohitO12-dev❌ GROUPINFRA\mattias.gradin
> 
> **GROUPINFRA\mattias.gradin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=4642) it.

@CodedByCoffee here is the CLA, could you please check if you can sign it? Let me know 😃

CodedByCoffee commented:
@mamtawardhani So now I have fixed all that you requested, please have a look when you have the time :)

mamtawardhani commented:
> @mamtawardhani So now I have fixed all that you requested, please have a look when you have the time :)

Hey @CodedByCoffee! 
 As it seems, all the commits are made by `GROUPINFRA\mattias.gradin` and none by `CodedByCoffee`. Hence, the CLA only wants the signature from `GROUPINFRA\mattias.gradin`. This most probably means that the commits are made locally by `GROUPINFRA\mattias.gradin` and the email address used for making them is not added to the `CodedByCoffee`GitHub account.
 
I would suggest you add the email address to your github account and also suggest visiting the `add the email address used for this commit to your account link in the CLAassistant` message that explains the process of doing this.

Please refer to this: [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user). and add the correct email id that was used to make the commits which is `GROUPINFRA\mattias.gradin``

CodedByCoffee commented:
> > @mamtawardhani So now I have fixed all that you requested, please have a look when you have the time :)
> 
> Hey @CodedByCoffee! As it seems, all the commits are made by `GROUPINFRA\mattias.gradin` and none by `CodedByCoffee`. Hence, the CLA only wants the signature from `GROUPINFRA\mattias.gradin`. This most probably means that the commits are made locally by `GROUPINFRA\mattias.gradin` and the email address used for making them is not added to the `CodedByCoffee`GitHub account.
> 
> I would suggest you add the email address to your github account and also suggest visiting the `add the email address used for this commit to your account link in the CLAassistant` message that explains the process of doing this.
> 
> Please refer to this: [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user). and add the correct email id that was used to make the commits which is `GROUPINFRA\mattias.gradin`

@mamtawardhani Now I have added my work email to my Github account and it seems the CLA is recognizing that I have done so 👍

mamtawardhani commented:
@cigar-galaxy82 please approve this PR.

mamtawardhani reviewed (approved):
Thank you @CodedByCoffee for thie entry! 😃 

It is ready to be merged! 🚀

CodedByCoffee commented:
> Thank you @CodedByCoffee for thie entry! 😃
> 
> It is ready to be merged! 🚀

@mamtawardhani Great 😃
